### PR TITLE
general educational fixes in some files

### PR DESCRIPTION
**Pull request by giridhar7632**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] All the files I changed are in the same world language, for example: only English changes, or only Chinese changes, etc.

### CODE REVIEW

File: docs/how-to-help-with-video-challenges.md [hunk 1]
```diff
@@ -71,11 +71,11 @@ You can find the markdown files for video challenges at the following locations
 
 Pick a challenge markdown file from the options above.
 
-### Skim through the video associated with the challenge and create a mutiple choice question
+### Skim through the video associated with the challenge and create a multiple choice question
 
-First, find the videoId.
+First, find the video Id.
```
Comment (ieahleen) [inline docs/how-to-help-with-video-challenges.md:76]: in this case it is a variable, so `videoId` should be correct correct.

```suggestion
First, find the videoId.
```

File: docs/how-to-help-with-video-challenges.md [hunk 2]
```diff
@@ -71,11 +71,11 @@ You can find the markdown files for video challenges at the following locations
...
 
-For example, in the following code from the header of a video challenge markdown file, the videoId is "nVAaxZ34khk". On GitHub, the information should be layed out in a table format.
+For example, in the following code from the header of a video challenge markdown file, the video Id is "nVAaxZ34khk". On GitHub, the information should be laid out in a table format.
```
Comment (ieahleen) [inline docs/how-to-help-with-video-challenges.md:78]:
```suggestion
For example, in the following code from the header of a video challenge markdown file, the videoId is "nVAaxZ34khk". On GitHub, the information should be laid out in a table format.
```

File: docs/how-to-help-with-video-challenges.md [hunk 3]
```diff
@@ -28,13 +28,13 @@ Challenge description text, in markdown
 
 ```html
 <div>
-  example code
+ example code
```
Comment (ieahleen) [inline docs/how-to-help-with-video-challenges.md:31]: indentation is correct in code formatting

```suggestion
 example code
```

File: docs/how-to-help-with-video-challenges.md (hunk 2 above)
Comment (RandellDawson) [reply docs/how-to-help-with-video-challenges.md:78]:
```suggestion
For example, in the following code from the header of a video challenge markdown file, the `videoId` is "nVAaxZ34khk". On GitHub, the information should be laid out in a table format.
```

File: docs/how-to-help-with-video-challenges.md (hunk 1 above)
Comment (nhcarrigan) [reply docs/how-to-help-with-video-challenges.md:76]:
```suggestion
First, find the `videoId`.
```

**Review by raisedadead — COMMENTED** [review]
There is a whole lot of whitespace changes, not sure if this is intentional. If yes would like some context.

**Comment by giridhar7632** [comment]
> There is a whole lot of whitespace changes, not sure if this is intentional. If yes would like some context.

Tbh it is unintentional. My text-editor messed up. I tried my best to fix the indentation in code blocks.

**Review by RandellDawson — CHANGES_REQUESTED** [review]
@giridhar7632  Please see my suggested changes.

**Review by raisedadead — CHANGES_REQUESTED** [review]
> Tbh it is unintentional. My text-editor messed up. I tried my best to fix the indentation in code blocks.

No worries, we would need you to undo those. If you need help with this let us know or reach us in the Contributors chat.

File: docs/how-to-open-a-pull-request.md [hunk 4]
```diff
@@ -92,61 +92,61 @@ When you are working on regular bugs and features on our development branch `mas
...
-      # example:
-      # git checkout feat/add-numpy-video-question
+        # example:
+        # git checkout feat/add-numpy-video-question
 
-      git checkout -b <backup-branch-name>
+        git checkout -b <backup-branch-name>
 
-      # example:
-      #  git checkout -b backup-feat/add-numpy-video-question
```
Comment (giridhar7632) [inline docs/how-to-open-a-pull-request.md:144]: I don't know why, but this is not rendering properly

File: docs/how-to-setup-freecodecamp-locally.md [hunk 5]
```diff
@@ -79,11 +79,11 @@ This is essential, as it allows you to work on your own copy of freeCodeCamp on
...
 
 <details>
-   <summary>
-      How to fork freeCodeCamp on GitHub (screenshot)
-   </summary>
-   <br>
-   <img src="https://raw.githubusercontent.com/freeCodeCamp/freeCodeCamp/master/docs/images/github/how-to-fork-freeCodeCamp.gif" alt="How to fork freeCodeCamp on GitHub">
+  <summary>
+     How to fork freeCodeCamp on GitHub (screenshot)
+  </summary>
```
Comment (giridhar7632) [inline docs/how-to-setup-freecodecamp-locally.md:84]: Some problem with `details` tag, not rendering properly
I tried but can't figure out what the problem is

File: docs/how-to-setup-freecodecamp-locally.md [hunk 6]
```diff
@@ -249,11 +249,11 @@ However, accessing the user portfolio page is a little tricky. In development, G
 Simply clicking the **"Preview Custom 404 Page"** button will forward you to the correct page.
 
 <details>
-   <summary>
-      How to sign in when working locally (screenshot)
-   </summary>
-   <br>
-   <img src="https://user-images.githubusercontent.com/29990697/71541249-f63cdf00-2923-11ea-8a85-cefb6f9c9977.gif" alt="How to sign in when working locally">
```
Comment (giridhar7632) [inline docs/how-to-setup-freecodecamp-locally.md:256]: same problem with `<details>`

**Comment by giridhar7632** [comment]
> No worries, we would need you to undo those. If you need help with this let us know or reach us in the Contributors chat.

I think I need help. I'm messing it more trying to fix it.

**Comment by raisedadead** [comment]
> I think I need help. I'm messing it more trying to fix it.

No worries, you should reach on the forum or the chat here: https://chat.freecodecamp.org/channel/contributors

**Comment by nhcarrigan** [comment]
@giridhar7632 if you are still having errors with resolving these conflicts and rendering bugs, let me know - I would be happy to assist.

**Comment by raisedadead** [comment]
@nhcarrigan I was hoping to undo the unintentional formatting and resolve the conflicts, but please feel free to beat me to it. I have been working on other infra stuff for the moment.

File: docs/how-to-open-a-pull-request.md (hunk 4 above)
Comment (nhcarrigan) [reply docs/how-to-open-a-pull-request.md]: This appears to render as expected now.

File: docs/how-to-setup-freecodecamp-locally.md (hunk 6 above)
Comment (nhcarrigan) [reply docs/how-to-setup-freecodecamp-locally.md]: Also renders.

**Comment by nhcarrigan** [comment]
@giridhar7632 I've gone ahead and given this a rebase, and reverted the spacing changes to allow for a cleaner `diff` tree. Also, it appears that the rendering issues are resolved. 🙂

File: docs/how-to-setup-freecodecamp-locally.md (hunk 5 above)
Comment (giridhar7632) [reply docs/how-to-setup-freecodecamp-locally.md]: Ok

File: docs/how-to-setup-freecodecamp-locally.md (hunk 6 above)
Comment (giridhar7632) [reply docs/how-to-setup-freecodecamp-locally.md]: I don't know why
but I'm just seeing the code itself
may be my editor problem 😅

**Comment by giridhar7632** [comment]
I tried to fix all the unintentional white space changes today 😁😅

**Comment by RandellDawson** [comment]
@giridhar7632 You should really be testing these changes locally by building the docs and previewing the files.  You have added some addition whitespace on lines that were originally blank lines.  @nhcarrigan already fixed the whitespace issues for you earlier.  @nhcarrigan I think we possibly need to revert @giridhar7632's last commit.

**Comment by giridhar7632** [comment]
Sorry 😅

**Review by raisedadead — APPROVED** [review]
Hi @RandellDawson I checked the final version on this PR looks good. I am going to go ahead and merge this in.

@giridhar7632 Thanks for making this PR. We look forward to more contributions in future.

@nhcarrigan thanks for the fixes and assistance.

**Pull request merged**